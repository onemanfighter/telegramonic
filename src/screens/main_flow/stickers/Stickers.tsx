const Stickers = () => {
  return (
    <div>
      <h1>Stickers</h1>
    </div>
  );
};

export default Stickers;
