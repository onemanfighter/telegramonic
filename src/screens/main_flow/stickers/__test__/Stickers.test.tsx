import { render } from '@testing-library/react';
import Stickers from '../Stickers';

describe('Stickers screen', () => {
  it('should render correctly', () => {
    const { container } = render(<Stickers />);

    expect(container).toMatchSnapshot();
  });
});
