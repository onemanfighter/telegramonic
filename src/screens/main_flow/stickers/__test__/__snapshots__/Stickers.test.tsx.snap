// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stickers screen should render correctly 1`] = `
<div>
  <div>
    <h1>
      Stickers
    </h1>
  </div>
</div>
`;
